src/DynamoCoreWpf/Interfaces/IWatchHandler.cs

```diff
@@ -220,7 +220,7 @@ private static string ToString(object obj)
         {
             return ReferenceEquals(obj, null)
                 ? Resources.NullString
-                : (obj is bool ? obj.ToString().ToLower() : obj.ToString());
+                : (obj is bool ? obj.ToString().ToLower() : obj is double? ((double)obj).ToString(CultureInfo.InvariantCulture) : obj.ToString());
```

Comment:
this is about the level of my comfort with nested ternary operators - any issue with using some if statements?

Reply:
Let me not put all of them in a single line to avoid confusion